client/src/pages/Login.tsx

```diff
@@ -37,10 +37,10 @@ const Login = () => {
   };
 
   return (
-    <div className="container border-2 w-5/10 h-5/10 flex justify-center items-center rounded-md flex-col">
+    <div className="container border-2 w-9/10 md:w-5/10 h-5/10 flex justify-center items-center rounded-md flex-col">
       <h1 className="text-lg">Login to Team Up</h1>
       <form
```

Comment:
ideally we can add some client side validation to the form input, for example, email field only accept email format input.

this can be done through `react-hook-form` and validation lib `zod`:
https://ui.shadcn.com/docs/components/form